js/gulpfile.js

```diff
@@ -185,44 +185,125 @@ gulp.task("test:common:generate", (cb) => {
     pump([gulp.src(["../scripts/Controller.ice"]), slice2js(), gulp.dest("test/Common")], cb);
 });
 
+// A rollup plug-in to resolve "net" module as a mockup file.
+
+const nodeMokups = ["net", "fs", "path"];
+function NodeMookupResolver() {
+    return {
+        name: "mookup-resolver",
+        async resolveId(source) {
+            if (nodeMokups.includes(source)) {
+                return path.join(__dirname, `test/Common/${source}.mockup.js`);
+            }
+            return null;
+        },
+    };
+}
+
+gulp.task("ice:bundle", async (cb) => {
+    let bundle = await rollup({
+        input: "src/index.js",
+        plugins: [NodeMookupResolver()],
```

Comment:
We use a mockup "net" module when building "ice" for the browser, this is a common strategy with WebPack too.